src/officehours_api/test_views.py

```diff
@@ -92,6 +97,29 @@ def test_cannot_change_backend_type_when_meeting_has_started(self):
             "Can't change backend_type once meeting is started!"
         )
 
+    def test_export_meeting_start_logs(self):
+        self.client.login(username='hosttwo', password='rohqtest')
+        # Start the meeting through the api to generate logs
+        response = self.client.post(f'/api/meetings/{self.meeting.id}/start')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.client.login(username='hostone', password='rohqtest')
+        # Start the meeting through the api to generate logs
+        response = self.client.get(f'/api/export_meeting_start_logs/')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # This may change if the output of the export changes
+        self.assertEqual(len(response.content), 343)
+
+    def test_export_meeting_start_logs_by_queue(self):
+        self.client.login(username='hosttwo', password='rohqtest')
+        # Start the meeting through the api to generate logs
+        response = self.client.post(f'/api/meetings/{self.meeting.id}/start')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response = self.client.get(f'/api/export_meeting_start_logs/{self.queue.id}/')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # This may change if the output of the export changes
+        self.assertEqual(len(response.content), 345)
```

Comment:
Tested with command `docker-compose run web python manage.py test officehours_api.test_views.MeetingTestCase`, and this test failed

======================================================================
FAIL: test_export_meeting_start_logs_by_queue (officehours_api.test_views.MeetingTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours_api/test_views.py", line 120, in test_export_meeting_start_logs_by_queue
    self.assertEqual(len(response.content), 345)
AssertionError: 291 != 345